torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1902,6 +1927,7 @@ def hint():
                         # vr is superset of c_vr
                         if not (c_vr.lower == r.lower and c_vr.upper == r.upper):
                             record_constraint_violation(lambda: (
+                                c,
```

Comment:
@avikchaudhuri fixed a typo here, so there'll be a merge conflict shortly. (This is supposed to be `c_vr` not `c`)